Mo & Joel - Targets workshop

drmowinckels opened the pull request.

drmowinckels commented:
@steffilazerte carpentries blog admin are wondering about the feasibility of having the post published next week? No rush, but I just wanted to convey their wishes.

steffilazerte commented on content/blog/2023-07-06-teaching-targets-with-penguins/index.en.md:
```diff
@@ -0,0 +1,93 @@
+This made a really nice dynamic between the two of us, and made the total experience even more enjoyable.
+
+The workshop was hosted by the [Digital Scholarship Center](https://www.ub.uio.no/english/libraries/dsc/) (DSC) at the University of Oslo, which also hosts the local [Carpentries community (Carpentry@UiO)](https://www.ub.uio.no/english/libraries/dsc/carpentry-uio/).
+The DSC took care of the important administrative tasks when hosting a workshop; setting up a website to announce the workshop, a sign-up solution (with waiting list), booking an accessible and suitable room, etc.
```
```suggestion
The DSC took care of the important administrative tasks required to host a workshop: setting up a website to announce the workshop, handling the sign-ups (including a waiting list), booking an accessible and suitable room, etc.
```

steffilazerte commented on content/blog/2023-07-06-teaching-targets-with-penguins/index.en.md:
```diff
@@ -0,0 +1,93 @@
+
+The workshop was hosted by the [Digital Scholarship Center](https://www.ub.uio.no/english/libraries/dsc/) (DSC) at the University of Oslo, which also hosts the local [Carpentries community (Carpentry@UiO)](https://www.ub.uio.no/english/libraries/dsc/carpentry-uio/).
+The DSC took care of the important administrative tasks when hosting a workshop; setting up a website to announce the workshop, a sign-up solution (with waiting list), booking an accessible and suitable room, etc.
+They additionally provided sticky-notes and extension cords for the room on the day.
```
```suggestion
Additionally, they provided sticky-notes and extension cords for the room on the day.
```

steffilazerte commented on content/blog/2023-07-06-teaching-targets-with-penguins/index.en.md:
```diff
@@ -0,0 +1,93 @@
+The workshop was hosted by the [Digital Scholarship Center](https://www.ub.uio.no/english/libraries/dsc/) (DSC) at the University of Oslo, which also hosts the local [Carpentries community (Carpentry@UiO)](https://www.ub.uio.no/english/libraries/dsc/carpentry-uio/).
+The DSC took care of the important administrative tasks when hosting a workshop; setting up a website to announce the workshop, a sign-up solution (with waiting list), booking an accessible and suitable room, etc.
+They additionally provided sticky-notes and extension cords for the room on the day.
+In short, organising the workshop was made very smooth by the expert and friendly staff of the DSC.
```
Wish I had a DSC for when I give workshops! 😆

drmowinckels replied:
right? They are awesome, they make the whole process so much easier for us instructors.

steffilazerte commented on content/blog/2023-07-06-teaching-targets-with-penguins/index.en.md:
```diff
@@ -0,0 +1,93 @@
+
+### Participants
+
+We had 19 participants sign up, and a little over half came (about normal no-show rate for DSC workshops). The participants covered a wide range of disciplines, from psychology to biology and humanities, and in every academic stage.  This varied group was perfect to test the workshop on!
```
```suggestion
We had 19 participants sign up, and more than half of those who signed up came (about normal no-show rate for DSC workshops). The participants covered a wide range of disciplines, from psychology to biology to humanities, and in every academic stage.  This diverse group was perfect to test the workshop on!
```

steffilazerte commented on content/blog/2023-07-06-teaching-targets-with-penguins/index.en.md:
I think that for the English post (and for now), the file is just `index.md` (cf Blog Guide section on [Multilingual posts](https://blogguide.ropensci.org/authortechnical.html#multilingual-posts)) @maelle, is that correct?

yabellini replied:
Yes, the name for the English is index.md :-)

steffilazerte commented on content/blog/2023-07-06-teaching-targets-with-penguins/index.en.md:
```diff
@@ -0,0 +1,93 @@
+
+Happy workflowing!
+
+![Joel and Mo after the workshop](mo-joel.jgp)
```
Use Hugo codes and alt text (cf [Blog Guide images](https://blogguide.ropensci.org/authortechnical.html#images---general))
```suggestion
{{< figure src = "mo-joel.jgp" alt = "Joel and Mo posing with grins after the workshop in front of a large projector screen showing the rOpenSci website behind them." caption = "Joel and Mo after the workshop" class = "center" >}}
```

steffilazerte reviewed (commented):
Hello @drmowinckels, @joelnitta!

Sorry for the delay, I've been away (and will be away Thurs/Fri), but had a chance to take a look today. 

Thank you for a lovely note on this workshop. It looks like a great resource to share and nice to have the experience shared here! I especially liked the mix of story (how it came about) and details to help others (logistics), all within a small useful post. Great!

I have some suggestions, some wording tweaks, some formatting tweaks. Nothing major, and most at your discretion (feel free to accept, reject or modify suggestions). 

The only ones that must be addressed are the logistical ones: 
- Update the date in the YAML key/folder name (happy to publish with the Carpentries next week, see my comment regarding date choices)
- Confirm the alt text I suggested for the image
- Consider the 'preface' if this is being cross-posted

If you have time to take a look by Monday July 17th (or at least one day before your chosen publication date), I'll be back to do a last quick scan and publish on the date you choose.

Just as a heads up: @drmowinckels I saw that you adjusted your name in the author file. I tweaked some other files (Champions mentor files) to make sure they're all using the same file (and pushed those changes here). 

Thanks again!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [x] publication date is ok
* [x] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [x] YAML field 'preface' is present if necessary

drmowinckels commented on content/blog/2023-07-06-teaching-targets-with-penguins/index.en.md:
```diff
@@ -0,0 +1,93 @@
+  - teaching
+  - meetings
+  - community
+---
```
@steffilazerte is it ok to add the preface key later, as I think the cross-post will happen after rOpenSci post, if I understand carpentry procedure correct.

drmowinckels replied:
preface added after confirmation from Carpentries.

drmowinckels commented:
ok @steffilazerte . I believe I have addressed all your comments. Let me know if I omitted anything. 
I've let carpentries blog admin know about publication date 20th of July, and asked about the cross-posting link. I'll let  you know once I hear back.

steffilazerte commented:
Hi @drmowinckels, thanks for your edits! I noticed that some of my comments were missed (this happens all the time, alas, as they're the "11 hidden comments" in the middle, above, which haven't been resolved. 

Would you have time to take a quick look at those outstanding comments?

steffilazerte commented on content/blog/2023-07-20-teaching-targets-with-penguins/index.md:
```diff
@@ -0,0 +1,96 @@
+---
+title: Teaching Targets with Penguins
```
Since the package name is really lowercase
```suggestion
title: Teaching targets with Penguins
```

drmowinckels commented:
Sorry for missing your other comments @steffilazerte 🙈 I hope I've resolved them all now. 

Mastodon post looks ok to me, thank you!

steffilazerte commented:
No worries, it happens all the time and I've been meaning to point that out when I do the review (but I keep forgetting) 🤦🏼 

Thanks, everything looks great and it's bee posted! The link to the Carpentries isn't up yet, but hopefully they'll post today so it isn't broken for too long! I'll hold off on the Mastodon post for a little while, as I'd hate to include a link to no where 😁 

Thanks again! 🙏🏼
